src/main/java/com/github/tomakehurst/wiremock/extension/Extensions.java

```diff
@@ -132,6 +135,14 @@ private void configureTemplating() {
     }
   }
 
+  private void configureWebhooks() {
+    final List<WebhookTransformer> webhookTransformers =
+        ofType(WebhookTransformer.class).values().stream().collect(Collectors.toUnmodifiableList());
+
+    final Webhooks webhooks = new Webhooks(webhookTransformers, options.getProxyTargetRules());
+    loadedExtensions.put(webhooks.getName(), webhooks);
```

Comment:
This would of course overwrite any existing extension of name `webhook`, but I guess you had thought of that and are OK with it.

Reply:
Yes, the idea is that you shouldn't have to do it yourself any more.

It takes the same network rules as the proxy from the options object and there's a new extension point for `WebhookTransformer` so no need to new it up yourself.